media/docs/cpp/xe_rearchitecture.md

```diff
@@ -290,6 +290,7 @@ Now that we have the basic thread mapping rule, let's apply it to a simple block
 An individual DPAS atom's A matrix follows the same pattern, with height ranging from 1 to 8, and width equal to 8 (tf32), 16 (f16/bf16), or 32 (s8/u8). The DPAS C matrix is also organized this way, except that its width is always 16.
 
 As a more complicated example, let's consider a 16-bit VNNI load, with height = 4, width = 16:
+
```

Comment:
Could you add the explanation of how to understand this representation which helped you? I think it would also be good to include (/link to) a good explanation of the VNNI format. Such as:
<img width="751" height="292" alt="image" src="https://github.com/user-attachments/assets/58ac9d9a-56a7-4f15-a1c2-c0cba47d6af3" />

Reply:
Thanks for reviewing!

Peter clarified that the numbers in subgroup view in the current documentation represent register offset. I added that information in the latest commit in this PR.

I added a link to https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc#example-2-16-bit-elements that describes VNNI layout.